<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoodTrackingView</class>
 <widget class="QWidget" name="MoodTrackingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>852</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>aakar</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGraphicsView" name="graphicsView_PM10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>331</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_CO">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>180</y>
     <width>341</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_SO2">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>180</y>
     <width>331</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_Mood">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>1111</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PM10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SO2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mood</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mood Tracking with Environmental Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
